server/db/types.go

```diff
@@ -0,0 +1,14 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package db
+
+// Account holds data returned by Accounts. Byte array fields in the database
+// are encoded as hex strings.
+type Account struct {
+	AccountID  string
+	Pubkey     string
+	FeeAddress string
+	FeeCoin    string
+	BrokenRule byte
```

Comment:
We haven't been able to settle on what idiomatic JSON is, but it definitely isn't fields *starting* with a capital letter.  Let's json tag these fields to be consistent with other instances of these values such as those in `dex/msgjson`.

I'm actually wavering on the types as well.  `account.AccountID` has a Stringer that JSON marshalling would use, as does `dex.Bytes`.  Do you foresee any issues with that?

Reply:
No! That makes much more sense.